data/news/platform/ocaml-5.0.beta1.md

```diff
@@ -92,9 +81,7 @@ In both cases, all available options can be listed with `opam search ocaml-optio
 ## Optional opam Alpha Repository
 
 During the beta release, if your dependencies are not yet compatible with OCaml 5.0.0, 
-you might want to check the alpha opam repository:
-
-https://github.com/kit-ty-kate/opam-alpha-repository
+you might want to check the [alpha opam repository](https://github.com/kit-ty-kate/opam-alpha-repository).
```

Comment:
It is not that useful to make this specific link clickable since I expect most people to only use this url through opam.